stable/datadog/README.md

```diff
@@ -164,3 +166,19 @@ datadog:
 To enable event collection, you will need to set the `datadog.leaderElection`, `datadog.collectEvents` and `rbac.create` options to `true`.
 
 Please read [the official documentation](https://docs.datadoghq.com/agent/basic_agent_usage/kubernetes/#event-collection) for more context.
+
+### Kubernetes Labels and Annotations
+
+To map Kubernetes pod labels and annotations to Datadog tags, provide a dictionary with kubernetes labels/annotations as keys and datadog tags as values:
+
+```yaml
+podAnnotationsAsTags:
+  cni.projectcalico.org/podIP: kube_podip
```

Comment:
We currently cannot recommend setting the pod IP as an autodiscovery tag, as this tag will change every time the pod restarts. For your use, you could prefix it with a `+` to only add it to the docker/kube metrics metrics and logs, see https://github.com/DataDog/datadog-agent/blob/f98440e80abd9d14b3b3dfb5b768d9437d320067/pkg/config/config_template.yaml#L478-L495

Would you be OK removing it from the example?